Resources/Prototypes/Entities/Structures/Power/apc.yml

```diff
@@ -27,7 +27,7 @@
     anchored: true
   - type: Sprite
     drawdepth: WallMountedItems
-    sprite: Structures/Power/apc.rsi
+    sprite: DeltaV/Structures/Power/apc.rsi #Delta V - Add sprite rotations
```

Comment:
Code comments should have leading spaces, "DeltaV" should be one word so it's easier to find using code search in the future.
`# DeltaV - Add sprite rotations`